tinyinst.cpp

```diff
@@ -1125,9 +1125,20 @@ void TinyInst::Init(int argc, char **argv) {
 
   std::list <char *> module_names;
   GetOptionAll("-instrument_module", argc, argv, &module_names);
-  for (auto iter = module_names.begin(); iter != module_names.end(); iter++) {
+#if defined __APPLE__ && defined ARM64
+  std::set <std::string> uniq_mod_names;
+  std::map<std::string, std::vector<std::string>> mod_grp = 

```

Comment:
Should we FATAL() if the resulting map is empty? This IIUC would indicate a parsing error.

Reply:
done.